### PR TITLE
Update links for Kubernetes yaml files

### PR DESCRIPTION
**Pull request by tkyi**
- Short-term fix for broken links in Cluster Setup page
- Added some more newlines for formatting issues

Relates to https://github.com/screwdriver-cd/guide/issues/35

### CODE REVIEW

File: docs/getting-started/cluster-setup.md [hunk 1]
```diff
@@ -97,24 +99,25 @@ Other environment variables can also be customized for Screwdriver. For a full l
 
 
 ## Deploy Screwdriver
-You can check out the [Screwdriver API repo](https://github.com/screwdriver-cd/screwdriver) and look in the `kubernetes` directory for service and deployment definitions for running the Screwdriver API. Both files are required to deploy Screwdriver.
+You can check out the [Screwdriver Kubernetes repo](https://github.com/screwdriver-cd/kubernetes) and for service and deployment definitions for running the Screwdriver API. Both files are required to deploy Screwdriver.
```
Comment (d2lam) [inline docs/getting-started/cluster-setup.md]: `and look for`

**Review by d2lam — CHANGES_REQUESTED** [review]

File: docs/getting-started/cluster-setup.md (hunk 1 above)
Comment (petey) [inline docs/getting-started/cluster-setup.md]: Which file(s) are service and which are deployment definitions? There are a lot of files in that repo.

Comment (tkyi) [reply docs/getting-started/cluster-setup.md]: Fixed!
Just dropped the `and`

Comment (tkyi) [reply docs/getting-started/cluster-setup.md]: Addressed!

Comment (petey) [reply docs/getting-started/cluster-setup.md]: My beef was with "Both files are required to deploy Screwdriver." This line makes me think there should be two files to look at, and not a multitude.

Comment (tkyi) [reply docs/getting-started/cluster-setup.md]: Ahh I see
Fixed! I just deleted that extra line. Service and Deployment used to be defined in separate files; in the new Kubernetes repo they are both defined in one file (api.yaml)

**Review by stjohnjohnson — APPROVED** [review]
Reasonable changes

**Review by petey — APPROVED** [review]
I'm cool with this.

blue_eyes_cat__belly_rub__no_emotion

**Review by d2lam — APPROVED** [review]

**Pull request merged**